src/components/ContactUs/Form.js

```diff
@@ -11,7 +11,7 @@ function Form() {
     emailjs
       .sendForm(
         'service_qozhrg7',
-        'emplate_akxu7oa',
+        'template_akxu7oa',
         form.current,
         'B2JkePBOzV_nhSmeO',
       )
```

Comment:
Also, in the function, when the emailing request is succesfull, it gives back "OK" as an alert. It does not look good at all. Can you change the alert to a notification? 

You can have a look at this library: https://www.npmjs.com/package/react-notifications

Other than that, let us show something relevant on that notification other than just "OK" for a better user experience.